### DISCUSSION TITLE
BDD Member

### DISCUSSION THREAD

**Discussion by DimaShulkin** [bdd100k/bdd100k#130] [General] [answered]
Hello,
the dataset can be used in the frame of BSD-3-Clause Licence for BDD members. How to be BDD members?

Thank you
Dima

**Comment by fyu** [comment, marked as answer]
The toolkit is released under the BSD license, while the data is released under its own license: https://doc.bdd100k.com/download.html. If you are interested in using BDD100K dataset for commercial purposes, you can send me an email (which I think you have already done :-).

**Reply by dineshsoudagar** [reply]
@fyu , I have sent a mail to your mail ID.

**Comment by dips4717** [comment]
@fyu  If we want to use a part of BDD for development/training of a commercial model, do we have to pay or,  is it a copyright agreement we have to sign on?
